docs/source/testing_roles.rst

```diff
@@ -35,6 +35,58 @@ be sufficient for the `molecule test` command to work.
 
 Alternatively you can test roles in with :ref:`AnsibleEE<testing with ansibleee>`.
 
+Creating a VM to run molecule
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The
+[edpm_libvirt](https://github.com/openstack-k8s-operators/edpm-ansible/tree/main/roles/edpm_libvirt)
+molecule senario uses the delegated driver to run against localhost.
+Thus, it's better to run its inside an ephemeral virtual machine as
+it cannot be run insider of a container.
+
+Boot a VM based on `CentOS-Stream-GenericCloud-9-latest.x86_64.qcow2`
+from https://cloud.centos.org/centos/9-stream/x86_64/images.
+
+Use [repo-setup](https://github.com/openstack-k8s-operators/repo-setup)
+to configure the DNF repositories of the VM.
+
+.. code-block:: console
+
+    pushd /tmp
+    curl -sL https://github.com/openstack-k8s-operators/repo-setup/archive/refs/heads/main.tar.gz | tar -xz
+    pushd repo-setup-main
+    python3 -m venv ./venv
+    PBR_VERSION=0.0.0 ./venv/bin/pip install ./
+    ./venv/bin/repo-setup current-podified-dev
+    popd
+
+Close the git repository on the VM.
+
+.. code-block:: console
+
+    dnf install -y git
+    git clone git@github.com:openstack-k8s-operators/edpm-ansible.git
+
+Prepare molecule.
+
+.. code-block:: console
+
+    pushd edpm-ansible
+    python3 -m venv molecule-venv
+    source molecule-venv/bin/activate
+    pip install --upgrade pip
+    pip install -r molecule-requirements.txt
+    popd
```

Comment:
so you may also need to install packages listed in bindep depending on the image you use.
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/bindep.txt

i.e. 

pip install bindep
bindep -b | xargs sudo dnf -y --setopt=install_weak_deps=False install

this may not be required for the nova/libvirt roles but it likely is for other roles

Reply:
OK, I'll add that. Thanks Sean.

Reply:
this block works as written.
althogh it would be better to do the bindep install after sorucing the venv on line
77 in case and of the deps in molecule-requiremets.txt need to compile c moudules